docs/pages/installation.mdx

```diff
@@ -903,11 +903,11 @@ If you wish to uninstall Teleport at any time, see our documentation on [Uninsta
 ## Next steps
 
 Now that you know how to install Teleport, you can enable access to all of your
-infrastructure. Get started with:
+infrastructure. Get started with enrolling resources in your Teleport cluster:
 
-- [Server Access](enroll-resources/server-access/introduction.mdx)
-- [Kubernetes Access](enroll-resources/kubernetes-access/introduction.mdx)
-- [Database Access](enroll-resources/database-access/database-access.mdx)
-- [Application Access](enroll-resources/application-access/introduction.mdx)
-- [Desktop Access](enroll-resources/desktop-access/introduction.mdx)
+- [Servers](enroll-resources/server-access/introduction.mdx)
```

Comment:
nit: in docs/pages/enroll-resources/machine-id/access-guides/access-guides.mdx "Servers" are referred to as "Linux servers", do we care if the two match?